universal-login-relayer/lib/utils/pendingExecution.ts

```diff
@@ -0,0 +1,71 @@
+import WalletContract from '@universal-login/contracts/build/Wallet.json';
+import {utils, ContractFactory, Contract, Wallet} from 'ethers';
+import {sortExecutionsByKey} from '../utils/utils';
+import {concatenateSignatures} from '@universal-login/contracts';
+import {Message} from '@universal-login/commons';
+
+export default class PendingExecution {
+  private wallet: Wallet;
+  private walletContract: Contract;
+  private collectedSignatures: any;
+  private tx: string;
+
+  constructor(walletAddress: string, wallet: Wallet) {
+    this.wallet = wallet;
+    this.walletContract = new Contract(walletAddress, WalletContract.interface, this.wallet);
+    this.collectedSignatures = [];
+    this.tx = '0x0';
```

Comment:
`tx` is a bit unclear. Should we use `this.transactionHash` instead?
We should also introduce separate boolean to indicate if transaction was executed, e.g. `isExecuted`